limesurvey/static/html/limesurvey.html

```diff
@@ -1,11 +1,15 @@
 <div class="limesurvey_block">
-    <ul>
-        <li>
-            <a href="{self.survey_url}" target="_blank" rel="noopener noreferrer" class="survey-url">{self.survey_url}</a>
-        </li>
-        <li>
-            <p>Access Code: <span class="access-code">{self.access_code}</span></p>
-        </li>
-    </ul>
-    <button class="button action-primary get-survey-button">Get Survey URL</button>
+    <p>Survey URL: <a href="{self.survey_url}" class="survey-url">{self.survey_url}</a></p>
+    <p>Access Code: <span class="access-code">{self.access_code}</span></p>
+    <button class="button action-primary get-survey-button">Get Survey</button>
+    <iframe
+        class="survey-iframe"
+        src="{self.survey_url}"
+        frameborder="0"
+        width="960"
+        height="569"
```

Comment:
why this? is there a way of the iframe to adjust itself without these values?

Reply:
Yes, I added CSS styles to adjust the iframe